src/label-for-moment/label-for-moment.test.js

```diff
@@ -0,0 +1,35 @@
+import labelForMoment from './label-for-moment';
+import moment from 'moment';
+import _ from 'lodash';
+
+describe('_.labelForMoment()', () => {
+  it('should return in 00:00am format if the moment is in today', () => {
+    var momentInstance = moment().hour(10).minute(0);
+
+    expect(labelForMoment(momentInstance)).to.equal('10:00 AM');
```

Comment:
10:00am - please work on your detail-orientedness - I mentioned lowercase am in a couple places specifically and indirectly as well.